model/src/main/java/oracle/kubernetes/weblogic/domain/v2/BaseConfiguration.java

```diff
@@ -194,12 +210,23 @@ void addPodAnnotations(String name, String value) {
     serverPod.addPodAnnotations(name, value);
   }
 
+  public Integer getRestartVersion() {
+    return restartVersion;
+  }
+
+  public abstract RestartVersion addEffectiveRestartVersion(RestartVersion restartVersion);
```

Comment:
EffectiveRestartVersion should not be on base configuration - it should be on ServerConfig